src/libraries/System.Private.CoreLib/src/System/IO/Strategies/FileStreamHelpers.Windows.cs

```diff
@@ -27,6 +28,10 @@ internal static class TaskSourceCodes
             internal const ulong ResultMask = ((ulong)uint.MaxValue) << 32;
         }
 
+        private const uint NT_ERROR_STATUS_DISK_FULL = 0xC000007F;
+        private const uint NT_ERROR_STATUS_FILE_TOO_LARGE = 0xC0000904;
+        private const uint NT_STATUS_INVALID_PARAMETER = 0xC000000D;
+
```

Comment:
I feel like these consts should live in the same file where `Interop.NtDll.CreateFile` lives.